### ISSUE TITLE
Inappropriate error message when updating kubeconfig of member cluster fails

### ISSUE THREAD

**Issue by wanjunlei** [kubesphere/kubesphere#4965]
**Describe the Bug**

Update the kubeconfig of a member cluster with the wrong kubeconfig, whether the content is wrong, the format is incorrect, or the kubeconfig is from another cluster, the suitable error message will appear the first time you click. After this, it will return an error:

```
couldn't get version/kind; json parse error: json: cannot unmarshal string into Go value of type struct { APIVersion string "json:\"apiVersion,omitempty\""; Kind string "json:\"kind,omitempty\"" }

```

For UI issues please also add a screenshot that shows the issue.

**Versions Used**
KubeSphere: ks3.3-rc.2
Kubernetes: (If KubeSphere installer used, you can skip this)

**Environment**
How many nodes and their hardware configuration:

For example: CentOS 7.5 / 3 masters:  8cpu/8g; 3 nodes: 8cpu/16g
(and other info are welcomed to help us debugging)

**How To Reproduce**

**Expected behavior**

**Comment by iawia002** [comment]
This is because when the update box is clicked a second time without closing it, the front-end will base64 encode the already base64 encoded kubeconfig again:

<img width="960" alt="WechatIMG42" src="https://user-images.githubusercontent.com/9134003/173488470-bffcc535-41d7-4f79-8152-d3ba821d4d10.png">

**Comment by wenxin-01** [comment]
Issue fixed in `v3.3.0-rc.3`
![image](https://user-images.githubusercontent.com/68640256/174697704-f34e4163-c6e4-4335-8997-069797760c2c.png)
